src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -354,11 +354,11 @@
   "entity.worker.inventoryfullchestfull": "Please make space in my chests/racks. I am not able to dump my inventory.",
 
   "entity.builder.messagebuildstart": ": I've started work on %s!",
-  "entity.builder.messagebuildcomplete": ": I've finished building %s!",
-  "entity.builder.messagebuildcomplete_manual": ": I've finished building %s. You can assign me another task now!",
+  "entity.builder.messagebuildcomplete": ": I've finished building %s at x=%s y=%s z=%s",
```

```suggestion
  "entity.builder.messagebuildcomplete": ": I've finished building the %s at %s %s %s!",
```
Not sure if we need the `xyz` there